addons/website_slides_survey/static/src/xml/website_slides_fullscreen.xml

```diff
@@ -2,11 +2,16 @@
 <templates>
 
     <t t-name="website.slides.fullscreen.certification">
-        <div class="justify-content-center align-self-center">
+        <div class="justify-content-center align-self-center" style="text-align:center;">
```

Comment:
(Forgot this one during the review, sorry for the spam)

We try to never write css in the "style" attribute but use bootstrap classes instead.
Here "text-center" should do the trick?